security/pkg/server/ca/authenticate/xfcc_authenticator_test.go

```diff
@@ -130,7 +138,8 @@ func TestXfccAuthenticator(t *testing.T) {
 			if len(tt.xfccHeader) > 0 {
 				md.Append(xfccparser.ForwardedClientCertHeader, tt.xfccHeader)
 			}
-			ctx := peer.NewContext(context.Background(), &peer.Peer{Addr: &net.IPAddr{IP: net.ParseIP("127.0.0.1").To4()}})
+			addr := net.TCPAddrFromAddrPort(netip.MustParseAddrPort("127.0.0.1:2301"))
```

Comment:
```suggestion
			addr := net.TCPAddrFromAddrPort(netip.MustParseAddrPort(tt.peer))
```

unless I am missing something

Reply:
There is tt.peer here. Earlier we had this net.ParseIP("127.0.0.1") and now it is changed to net.TCPAddrFromAddrPort(netip.MustParseAddrPort("127.0.0.1:2301"))

Reply:
yes it is hard coded. Let us merge this and I will improve test cases to move it out to test params? If you are OK with that I will remove the `do-not-merge'  label.